### PR TITLE
Remove blanket internet connection status (EXPOSUREAPP-1572) 

### PR DESCRIPTION
**Pull request by Oliver-Zimmerman**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [x] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) - Pull Requests that are not linked to an issue with you as an assignee will be closed, except for minor fixes for typos or grammar mistakes in *documentation or code comments*.
* [ ] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)
* [x] Make sure that your PR does not contain changes in text files, therefore the PR must not contain changes in values/strings/* and / or assets/* (see issue #332 for further information)
* [x] Make sure that your PR does not contain compiled sources (already set by the default .gitignore) and / or binary files

## Description
Removed the blanket internet connection status:

Removed the connection state from TracingStatusHelper
Adjusted formatters which use the adjusted TracingStatusHelper
Removed connection logic from various fragments (SettingsTracingFragment, etc.)
Adjusted XML views (Settings Fragment: no connection card, etc.)
Adjusted automated tests that previously included the connection state.

Test:
Use application with mobile data and wifi turned off, exposure logging will still be active (visible in Main fragment and Settings Fragment). Separate dialogs for things like TAN and QR code scanning, where an internet connection is required, will mention there is no internet connection. 

Closes JIRA task EXPOSUREAPP-1572

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -3,7 +3,16 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
```
Comment (jakobmoellersap) [inline .idea/codeStyles/Project.xml:6]: Imported here by accident?

**Review by jakobmoellerdev — CHANGES_REQUESTED** [review]
Remove Project.xml

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/util/formatter/FormatterSettingsHelperTest.kt [hunk 2]
```diff
@@ -210,7 +198,6 @@ class FormatterSettingsHelperTest {
     private fun formatTracingIconBase(
         bTracing: Boolean,
         bBluetooth: Boolean,
-        bConnection: Boolean,
         bLocation: Boolean
     ) {
         every { context.getDrawable(R.drawable.ic_settings_tracing_bluetooth_inactive) } returns drawable
```
Comment (MertSafter) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/util/formatter/FormatterSettingsHelperTest.kt:203]: every { context.getDrawable(R.drawable.ic_settings_tracing_connection_inactive) } returns drawable

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/util/formatter/FormatterSettingsHelperTest.kt [hunk 3]
```diff
@@ -219,7 +206,7 @@ class FormatterSettingsHelperTest {
         every { context.getDrawable(R.drawable.ic_settings_tracing_inactive) } returns drawable
```
Comment (MertSafter) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/util/formatter/FormatterSettingsHelperTest.kt:206]:
```
        every { context.getDrawable(R.drawable.ic_settings_tracing_connection_inactive) } returns drawable
```
I think line 204 can be deleted as well.

**Review by jakobmoellerdev — APPROVED** [review]

**Pull request merged**